cdap-docs/developers-manual/source/getting-started/standalone/docker.rst

```diff
@@ -70,11 +70,10 @@ Kitematic and then download, start, and connect to an instance of CDAP.
         :width: 8in
         :align: center
 
-
 #. This shows that the CDAP instance is listening on the internal port ``9999`` within the
    Docker instance, while the Docker instance exposes that port on the external IP address
```

Comment:
s/instance/container/
s/instance/host/
s/exposes/proxies/
s/external/Virtual Machine/

A Docker container EXPOSEs ports, but when they're available outside the container, they've been proxied from the Docker host. In this case, the Docker host is the virtual machine.